devsetup/scripts/standalone.sh

```diff
@@ -23,7 +23,8 @@ IP="192.168.122.${IP_ADRESS_SUFFIX}"
 OUTPUT_DIR=${OUTPUT_DIR:-"${SCRIPTPATH}/../../out/edpm/"}
 SSH_KEY_FILE=${SSH_KEY_FILE:-"${OUTPUT_DIR}/ansibleee-ssh-key-id_rsa"}
 SSH_OPT="-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i $SSH_KEY_FILE"
-CMDS_FILE=${CMDS_FILE:-"/tmp/standalone_repos"}
+REPO_SETUP_CMDS=${REPO_SETUP_CMDS:-"/tmp/standalone_repos"}
+CMDS_FILE=${CMDS_FILE:-"/tmp/standalone_cmds"}
 CLEANUP_DIR_CMD=${CLEANUP_DIR_CMD:-"rm -Rf"}
 
```

Comment:
probably safer to also default SKIP_TRIPLEO_REPOS to "" here too
will do on next update

```

SKIP_TRIPLEO_REPOS=${SKIP_TRIPLEO_REPOS:=""}
```

Reply:
actually default 'false'